src/vm/win32threadpool.h

```diff
@@ -1022,12 +1022,12 @@ class ThreadpoolMgr
         {
             LIMITED_METHOD_CONTRACT;
 
-	    if (CPUGroupInfo::CanEnableGCCPUGroups() && CPUGroupInfo::CanEnableThreadUseAllCpuGroups())
-            return pRecycledListPerProcessor[CPUGroupInfo::CalculateCurrentProcessorNumber()][memType];
-        else
-            // Turns out GetCurrentProcessorNumber can return a value greater than the number of processors reported by
-            // GetSystemInfo, if we're running in WOW64 on a machine with >32 processors.
-        	return pRecycledListPerProcessor[GetCurrentProcessorNumber()%NumberOfProcessors][memType];
+	        if (CPUGroupInfo::CanEnableGCCPUGroups() && CPUGroupInfo::CanEnableThreadUseAllCpuGroups())
```

Comment:
actually I just checked in the editor and this lines up correctly. it's the previous code that was misaligned.